admin/worker/run_autoscaler.go

```diff
@@ -18,10 +20,23 @@ func (w *Worker) runAutoscaler(ctx context.Context) error {
 	}
 
 	for _, rec := range recs {
-		// TODO: Add autoscaling logic based on the recommendation here.
-		// Consider checking rec.UpdatedOn to avoid making autoscaling decision on stale data!
-		w.logger.Info("autoscaler recommendation", zap.String("project_id", rec.ProjectID), zap.Int("recommended_slots", rec.RecommendedSlots))
-		break
+		duration := time.Since(rec.UpdatedOn)
+		if duration < 24*time.Hour {
+			w.logger.Debug("skipping autoscaler: the project has been scaled recently", zap.String("project_id", rec.ProjectID), zap.Time("project_updated_on", rec.UpdatedOn))
+			break
+		}
+
+		opts := &database.UpdateProjectOptions{
+			ProdSlots: rec.RecommendedSlots,
+		}
+
+		proj, err := w.admin.DB.UpdateProject(ctx, rec.ProjectID, opts)
```

Comment:
TODO:
Consider using admin client `UpdateProject` method instead of direct DB update since we may also want to:
- trigger restart the project
- enable observability for project update